gemsfx/src/main/java/com/dlsc/gemsfx/templatepane/TemplatePaneSkin.java

```diff
@@ -94,60 +94,78 @@ protected void layoutChildren(double contentX, double contentY, double contentWi
 		currentTileBounds.forEach((tile, bounds) -> tile.resize(bounds.getMinX(), bounds.getMinY(), bounds.getWidth(), bounds.getHeight()));
 	}
 
+        /*
+        AR: Es konnten negative Werte für contentHeight / contentWidth entstehen
```

Comment:
No German comments please. Also no need to prefix with your initials.

Reply:
OK, removed